root/report/index.tt

```diff
@@ -100,7 +100,8 @@
             <li><a href="[% c.uri_for_action('/report/show', 'RecordingsWithVaryingTrackLengths') %]">[% l('Recordings with varying track times') %]</a></li>
             <li><a href="[% c.uri_for_action('/report/show', 'DeprecatedRelationshipRecordings') %]">[% l('Recordings with deprecated relationships') %]</a></li>
             <li><a href="[% c.uri_for_action('/report/show', 'AnnotationsRecordings') %]">[% l('Recordings with annotations') %]</a></li>
-            <li><a href="[% c.uri_for_action('/report/show', 'NotVariousArtists') %]">[% l('Recordings incorrectly linked to Various Artists') %]</a></li>
+            <li><a href="[% c.uri_for_action('/report/show', 'RecordingsNotVA') %]">[% l('Recordings linked to Various Artists without "Various Artists" as the artist credit') %]</a></li>
+            <li><a href="[% c.uri_for_action('/report/show', 'RecordingsNoVALink') %]">[% l('Recordings not linked to Various Artists but with "Various Artists" as the artist credit') %]</a></li>
```

Comment:
Since these reports are basically the same other than checking whether the credits/links differ, I wonder if "RecordingsWithWrongVACredit" and "RecordingsWithWrongVALink" would be better.